src/java/picard/analysis/directed/TargetedPcrMetrics.java

```diff
@@ -17,79 +17,79 @@
     public long PF_READS;
 
-    /** THe number of bases in the SAM or BAM file to be examined */
+    /** The total number of bases within the PF_READS of the SAM or BAM file to be examined */
     public long PF_BASES;
 
-    /** The number of PF reads that are not marked as duplicates. */
+    /** The number of PF_READS that were not marked as sample or optical duplicates. */
     public long PF_UNIQUE_READS;
 
-    /** PF reads / total reads.  The percent of reads passing filter. */
+    /** The fraction of reads passing the vendor's filter (PF_READS/TOTAL_READS). */
     public double PCT_PF_READS;
 
-    /** PF Unique Reads / Total Reads. */
+    /** The fraction of PF_UNIQUE_READS from the total number of PF_READS (PF_UNIQUE_READS/PF_READS) */
     public double PCT_PF_UQ_READS;
 
-    /** The number of PF unique reads that are aligned with mapping score > 0 to the reference genome. */
+    /** The total number of PF_UNIQUE_READS that align to the reference genome with mapping scores > 0 */
     public long PF_UQ_READS_ALIGNED;
 
-    /** Tracks the number of read pairs that we see that are PF (used to calculate library size) */
+    /** Tracks the number of observed read pairs that pass the vendor's filter (used to calculate library size) */
     public long PF_SELECTED_PAIRS;
 
-    /** Tracks the number of unique PF reads pairs we see (used to calc library size) */
+    /** Tracks the number of read pairs that pass the vendor's filter that do not have any observed duplicate reads (used to calc library size) */
     public long PF_SELECTED_UNIQUE_PAIRS;
 
-    /** PF Reads Aligned / PF Reads. */
+    /** Fraction of PF_READS that lack sample or optical duplicates and align to the reference genome (PF_UQ_READS_ALIGNED/PF_UNIQUE_READS) */
     public double PCT_PF_UQ_READS_ALIGNED;
 
-    /** The number of PF unique bases that are aligned with mapping score > 0 to the reference genome. */
+    /** The total number of bases from PF_READS that align to the reference genome with mapping score > 0 */
     public long PF_BASES_ALIGNED;
 
-    /** The number of PF unique bases that are aligned with mapping score > 0 to the reference genome. */
+    /** The total number of bases from unique PF_READS that align to the reference genome and have a mapping score > 0 */
     public long PF_UQ_BASES_ALIGNED;
 
-    /** The number of PF aligned amplified that mapped to an amplified region of the genome. */
+    /** The total number of bases from aligned reads that pass the vendor's filter (PF_BASES_ALIGNED) that map to an amplified region of the genome. */
     public long ON_AMPLICON_BASES;
 
-    /** The number of PF aligned bases that mapped to within a fixed interval of an amplified region, but not on a baited region. */
+    /** The total number of bases from aligned reads that have passed the vendor's filter (PF_BASES_ALIGNED), that map to within a fixed interval of an amplified region, but not on a baited region. */
```

Comment:
This is not true. IntelliJ doesn't understand reflection...take a look
at picard/analysis/directed/TargetedPcrMetricsCollector.java:87

and you'll see that there's a renaming that is happening in a way that
IntelliJ is blind to it. Since the doc generation is also done via
reflection, it looks as though nothing looks at this code...but it isn't
true. Please do update these metrics.

On Tue, May 31, 2016 at 5:24 PM, David E. Kling <notifications@github.com>
wrote:

> In src/java/picard/analysis/directed/TargetedPcrMetrics.java
> <https://github.com/broadinstitute/picard/pull/549#discussion_r65265576>:
>
> >      public long ON_AMPLICON_BASES;
> >
> > -    /** The number of PF aligned bases that mapped to within a fixed interval of an amplified region, but not on a baited region. */
> > +    /** The total number of bases from aligned reads that have passed the vendor's filter (PF_BASES_ALIGNED), that map to within a fixed interval of an amplified region, but not on a baited region. */
>
> Note, I didn't make all the changes for TargetedPcrMetrics.java, because
> IntelliJ indicates that many of these are not used. Happy to do them though
> if you like. Just let me know.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/broadinstitute/picard/pull/549/files/a4c48fd5af9c8472ea9d4826cc6a0c6175598e61#r65265576>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACnk0hQ7yWILOLnHCKMsCq6xSaObdH71ks5qHKb3gaJpZM4IowZ8>
> .
>